plugins/show-language/prism-show-language.js

```diff
@@ -57,6 +57,7 @@
 		"dockerfile": "Docker",
 		"ebnf": "Extended Backus–Naur form",
 		"ejs": "EJS",
+		"eta": "EJS",
```

Comment:
This isn't good. It should be `ETA`.
That's a bug in Prism's build. I'll investigate.

Reply:
Will be fixed by #2283. Let's merge this PR!